### DISCUSSION TITLE
Videos - downloading far fewer

### DISCUSSION THREAD

**Discussion by tobyni** [TumblThreeApp/TumblThree#366] [Q&A (Question & Answer)]
I've only recently starting using TumblThree again, a lots changed, but the latest update seems to be working well, without errors.
However, starting fresh, I'm getting some strange results from a couple of blogs... The details pane is saying that it has downloaded only 58 out of 702 videos, with 20 duplicates.  (Blog NSFW, so not posting)

I've tried API and SVC, as well as reducing posts per page (vaguely remember that was a workaround a while back), but nothing increases the number of videos being downloaded.  I should say the same thing doesn't happen for images or audio files, they're all accounted for.

Am i missing something obvious?

**Comment by thomas694** [comment]
It looks like TumblThree has problems with the videos in these blogs, either because of NSFW or for other reasons. You may take a look into the log file that is created in the newest version, it's level of verbosity can be configured in general settings. Maybe you find a hint.

Does anyone else have this problem, or even have an example blog that shows these problems and can be posted here?

**Comment by tobyni** [comment]
You're right, I just tried **justcatposts**.tumblr.com and managed to snag around 95% of the videos, as opposed to just 10% on some of the NSFW blogs, so it must be an overlay or warning blocking them.  I doubt there'll be much to do to remedy that.

With regards the logs, I've verbose logging on but can't find an actual log file anywhere.

**Comment by thomas694** [comment]
The log file should be in %LOCALAPPDATA%\TumblThree or inside the program folder for portable mode.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
